analytics/spark/src/main/java/mil/nga/giat/geowave/analytic/spark/sparksql/udf/GeomWithinDistance.java

```diff
@@ -34,6 +35,7 @@ public void setRadius(
 	public boolean apply(
 			Geometry geom1,
 			Geometry geom2 ) {
-		return geom1.distance(geom2) <= radius;
+		return geom1.distance(
```

Comment:
formatting change, can you run `mvn` on the commandline to make sure this is right?  (do you run the formatter in eclipse IDE because that may be different?)